### PR TITLE
feat: reload page when clicking on create request

### PR DESCRIPTION
**Pull request by jjramirezn**
To permit easy creation of new request after creating one

### CODE REVIEW

File: src/components/Global/Header/index.tsx [hunk 1]
```diff
@@ -228,7 +228,12 @@ const MenuLinks = () => {
             <NavLink href={'/send'} onClick={handleClick}>
                 <Text display="block">send</Text>
             </NavLink>
-            <NavLink href={'/request/create'}>
+            <NavLink
+                href={'/request/create'}
+                onClick={() => {
+                    if (window.location.pathname == '/request/create') window.location.reload()
+                }}
+            >
```
Comment (Hugo0) [inline src/components/Global/Header/index.tsx]: TODO: nice - I think this is the desired behavior. For consistency, can we have the same behaviour across all pages? So send, request, cashout.

Generally when submitting a PR we should think holistically - else we get fragmented behaviour. Even though the ticket was submitted only for request links navbar, its valid across other items, so the issue fix shouldn't just tackle the complaint, but ideally tackle whole issue holistically.  And if not, it should at least make mention of this wider issue

a very similar thing was raised in this PR from @panosfilianos: https://github.com/peanutprotocol/peanut-ui/pull/426#pullrequestreview-2356706285

**Review by Hugo0 — CHANGES_REQUESTED** [review]

File: src/components/Global/Header/index.tsx (hunk 1 above)
Comment (jjramirezn) [reply src/components/Global/Header/index.tsx]: great point, implementing

File: src/components/Global/Header/index.tsx [hunk 2]
```diff
@@ -115,6 +115,37 @@ const MenuToggle = ({ toggle, isOpen }: { toggle: () => void; isOpen: boolean })
...
+            if (window?.location.pathname === route) {
+                // Force a hard reload of the current page
+                window.location.reload()
+            } else {
+                // For different routes, use router.push()
+                router.push(route)
+            }
+        },
+        [router, route]
+    )
```
Comment (Hugo0) [inline src/components/Global/Header/index.tsx:133]: praise: nice! Way cleaner :)

File: src/components/Global/Header/index.tsx [hunk 3]
```diff
@@ -115,6 +115,37 @@ const MenuToggle = ({ toggle, isOpen }: { toggle: () => void; isOpen: boolean })
     )
 }
 
+const MenuLink = ({ route, title, isBeta = false }: { route: string; title: string; isBeta?: boolean }) => {
```
Comment (Hugo0) [inline src/components/Global/Header/index.tsx:118]: thought: in the future we might also want some other properties (e.g. isNew or hasNotification or similar)

**Review by Hugo0 — APPROVED** [review]
v nice!

**Pull request merged**